personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -36,46 +71,30 @@ function PersonalityTestXBlockStudent(runtime, element) {
                         if (resp.success) {
                             studentAnswers = JSON.parse(resp['answers']);
                         }
-
                         var myForm = $('.personality-test-form', element);
                         var mainDiv = $('.personality-test-form-table', element);
 
                         var list = document.createElement('ol');
                         var i = 0;
+                        var ii = 0;
                         questions.forEach(function (question) {
-                            var questionInList = document.createElement('li');
-                            var spanQuestion = document.createElement('div');
-                            spanQuestion.className = 'question-div';
-
-                            var select = document.createElement('select');
-                            var opt = document.createElement('option');
-                            opt.text = '';
-                            opt.value = '';
-                            select.add(opt);
-                            var studentAnswer = '';
-                            if (studentAnswers !== '') {
-                                studentAnswer = getAnswersValue(studentAnswers, question['id']);
+                            if (question.type === 'group') {
+                                var b = document.createElement('b');
+                                b.appendChild(document.createTextNode(++i + '.' + question.description));
+                                list.appendChild(b);
+                                var subList = document.createElement('ol');
+                                question.questions.forEach(function (item) {
+                                    ++ii;
+                                    addQuestion(item, subList, i + '.' + ii, studentAnswers);
```

Comment:
what is this `i + '.' + ii`? from what I see, `addQuestion`’s third argument is named `count`, this does not look like a number to me.